awx/ui/src/screens/InstanceGroup/shared/InstanceGroupForm.js

```diff
@@ -1,49 +1,25 @@
 import React from 'react';
 import { func, shape } from 'prop-types';
-import { Formik, useField } from 'formik';
+import { Formik } from 'formik';
 
 import { t } from '@lingui/macro';
 import { Form } from '@patternfly/react-core';
 
 import FormField, { FormSubmitError } from 'components/FormField';
 import FormActionGroup from 'components/FormActionGroup';
-import {
-  combine,
-  required,
-  protectedResourceName,
-  minMaxValue,
-} from 'util/validators';
+import { required, minMaxValue } from 'util/validators';
 import { FormColumnLayout } from 'components/FormLayout';
 
-function InstanceGroupFormFields({ defaultControlPlane, defaultExecution }) {
-  const [, { initialValue }] = useField('name');
-  const isProtected =
-    initialValue === `${defaultControlPlane}` ||
-    initialValue === `${defaultExecution}`;
-
-  const validators = combine([
-    required(null),
-    protectedResourceName(
-      t`This is a protected name for Instance Groups. Please use a different name.`,
-      [defaultControlPlane, defaultExecution]
-    ),
-  ]);
-
+function InstanceGroupFormFields() {
   return (
     <>
       <FormField
         name="name"
-        helperText={
-          isProtected
```

Comment:
I wish we didn't have to remove this but I think the refactor you've done here outweighs this little bit of lost UX.  I know you've done a lot of investigating into how to keep this and short of making some api changes that the api group doesn't support (for valid reason) I'm not real sure what else we can do.